modules/apps/foundation/frontend-theme/responsive-boat-theme/src/css/_custom.scss

```diff
@@ -0,0 +1,290 @@
+
+$mobileNavExpandTime: 0.4s;
+$navLinkFadeTime: 0.5s;
+$navBtnPadding: 20px;
+$navBtnHeight: 40px;
+
+/* All styling is nested within #wrapper selector to avoid accidentally styling the liferay menus */
+
+#wrapper {
+	background-attachment: fixed; 		// background stays fixed while scrolling
+	background-color: #A9A9A9; 			// default background when image is not loaded
+	background-image: url("../images/background.jpg");
+	background-position-x: center;
+	background-size: cover;
+	padding: 0; 						// eliminate white spaces on edges of webpage
+	text-align: center; 				// default text-alignment to center all child texts and elements
+
+	#banner {
+		background: rgba(0, 0, 0, 0.5);
+		color: #FFF;
+		padding-bottom: 500px;			// add space below navbar to copy the Wordpress theme
+	}
+
+	/* ---------- NAVIGATION SECTION ---------- */
+
+	#navigation {
+		background: rgba(0, 0, 0, 0.25);
+		position: fixed;
+		text-transform: uppercase;
+		width: 100%;
+		z-index: 1; 					// found that some portlet content overlaps the navigation bar unless z-index is set to 1 or higher
+
+		div.nav-btn {
+			display: none; 				// hide mobile nav-button until screen width is small enough
+		}
+
+		div, ul, .navbar, .portlet {
+			border-width: 0; 			// eliminate unnecessary borders that just make navbar taller (that were set outside of my css)
+		}
+
+		.portlet-topper { 				// restyle portlet topper for navbar
+			border-radius: 4px;
+			bottom: 0;
+			top: unset;
+			width: 200px;
+
+			span {
+				line-height: 32px;
+			}
+		}
+
+		.portlet:hover, .portlet:focus {
+			> .portlet-topper {
+				transform: translateY(100%); // change direction that portlet topper floats
+			}
+		}
+
+		.navbar, .portlet, ul {
+			margin: 0; 					// eliminate unnecessary and pesky margins in navbar (also set outside of my css)
+		}
+
+		.portlet-title-text { display: none; } // hide portlet-title (used to be hidden until I changed it to an h1 in the template)
+
+		ul.nav {
+			float: none;
+			list-style: none; 			// get rid of more unnecessary styling
+
+			li { 						// arrange nav-links horizontally
+				display: inline-block;
+				float: none;
+				padding: 20px 15px;
+
+				a {
+					background: none;
+					color: inherit;
+					display: block;
+					padding: 10px 0;
+					text-decoration: none;
+					transition: color $navLinkFadeTime;
+
+					&:hover {
+						color: #FFDD00;
+						transition: color $navLinkFadeTime;
+					}
+				}
+			}
+		}
+
+		&.scrolled { 					// JS-toggled class that reorganizes navbar links and darkens navbar
+			@include media-query(sm) {	// prevent this from applying to navbar when it's styled for mobile devices
```

Comment:
Since this CSS block contains only things on mobile, we should switch the CSS block, so the conditional is wrapping the CSS selector.